rebasehelper/specfile.py

```diff
@@ -410,9 +389,13 @@ def write_updated_patches(self, patches, disable_inapplicable):
                     patch_inapplicable = [x for x in patches['inapplicable'] if patch_name in x]
                 else:
                     patch_inapplicable = None
-                if check_not_applied or patch_removed:
+                if check_not_applied:
                     self.spec_content[index] = '#{0} {1}\n'.format(' '.join(fields[:-1]),
                                                                    os.path.basename(patch_name))
```

Comment:
Well it is questionable, as that information might be gathered from the changes patch. In any case, rebased spec file in that state will probably not be used most of the time, as another interactive rebase will need to be done. Do you agree?